src/msg/async/ProtocolV1.cc

```diff
@@ -315,6 +315,17 @@ void ProtocolV1::write_event() {
     auto start = ceph::mono_clock::now();
     bool more;
     do {
+
+      if (connection->outgoing_bl.length() > 0) {
+	r = connection->_try_send();
+	if (r < 0) {
+	  ldout(cct, 1) << __func__ << " pre-flush outgoing_bl fail." << dendl;
```

Comment:
`AsyncConnection::_try_send()` already prints logging message when the underlying socket fails to send the buffer list, probably we can just drop this error message and put something like:
```c++
if (r = connection->_try_send(); r != 0) {
  // either fails to send or not all queued buffer is sent
  break;
}
```

Reply:
sure, good suggestions.
already done with latest version